### PR TITLE
fix broken getCurrentPosition method

### LINKED ISSUE #277: getCurrentPosition is broken in ios
I was getting this error 
flutter: error MissingPluginException(No implementation found for method getCurrentPosition on channel xyz.luan/audioplayers)
but audioplayer was working fine in previous versions, so I took a look at commits and found this https://github.com/luanpotter/audioplayers/commit/8ed42bc8598ca99975e5726ba0d158579c474fcc

### PR DESCRIPTION
**Pull request by bardiarastin**
fixes #277 
I've tested this on my app and it's working perfectly.

### CODE REVIEW

**Comment by bardiarastin** [comment]
@erickzanardo would you please take a look at this? my app is broken because this method was deleted :(

**Comment by bardiarastin** [comment]
@erickzanardo I've added a new option in cafa960 (and docs in ad6da0e) for users to be able to set playerId when creating a new instance of AudioPlayer, this is because I'm using this lib for a music app so I don't need to play multiple audio files simultaneously and I'm ending up with multiple bugs cause I'm not able to use it to play one audio file at a specific time, this is not breaking anything and it's totally backward compatible, please take a look at it and tell me what do you think.

File: ios/Classes/AudioplayersPlugin.m [hunk 1]
```diff
@@ -449,6 +455,15 @@ -(int) getDuration: (NSString *) playerId {
     return mseconds;
 }
 
+-(int) getCurrentPosition: (NSString *) playerId {
+    NSMutableDictionary * playerInfo = players[playerId];
+    AVPlayer *player = playerInfo[@"player"];
+
+    CMTime duration = [player currentTime];
+    int mseconds= CMTimeGetSeconds(duration)*1000;
```
Comment (luanpotter) [inline ios/Classes/AudioplayersPlugin.m]: `return CMTimeGetSeconds(duration) * 1000;`

**Comment by bardiarastin** [comment]
hi @luanpotter, so much thanks for quick review, I've applied your requested changes, as I said I'm using this lib for a music app that we guess it's going to have at least a few million users, I mentioned @erickzanardo cause I taught you're a little busy, sorry about that.

we can launch whenever #232 lands and @subhash279 finishes his work on the notification bar, it would be great if you could spend more time on PR's and issues, I appreciate that.

**Pull request merged**